vital_site/vital/views/security_view.py

```diff
@@ -322,3 +322,39 @@ def index(request):
     else:
         logger.debug('user is admin')
 
+def release_vm(request):
+    logger.debug("in releaseVM")
+    error_message = ''
+
+    try:
+        if request.method == 'GET':
+            # Get the VM ID in request
+	    api_key = request.GET['api_key']
```

Comment:
rather pass back user_id and password back from xen_api and authenticate. Using the security key is less secure